hbase_sql/src/main/java/com/splicemachine/hbase/BackupUtils.java

```diff
@@ -231,7 +224,7 @@ public static boolean shouldCaptureIncrementalChanges(FileSystem fs,Path rootDir
                             String path = spliceBackupPath + "/" + backupId;
                             byte[] data = zooKeeper.getData(path, false, null);
                             BackupJobStatus status = BackupJobStatus.parseFrom(data);
-                            if (status.getScope() == BackupScope.DATABASE) {
+                            if (status.getScope() == BackupJobStatus.BackupScope.DATABASE.DATABASE) {
```

Comment:
Please remove the extra DATABASE in "BackupJobStatus.BackupScope.DATABASE.DATABASE"